### DISCUSSION TITLE
[FEATURE REQ] Ability to add category ID with Torznab Feeds

### DISCUSSION THREAD

**Discussion by bilyboy785** [autobrr/autobrr#1693] [Ideas]
**Version**
v1.43.0

**Describe the bug**
Add a Torznab indexer with query parameters like "cat" like : `http://prowlarr:9696/8/api?cat=102140`
Enable the feed
When force run, and see the latest result content, the category is not used for the search

**Expected behavior**
Expected to see results only for the specified category.

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser Chrome
 - Version 126.0.6478.127

**Comment by zze0s** [comment]
@bilyboy785 Hey! This is finally being implemented in https://github.com/autobrr/autobrr/pull/2323
